UnityProject/WhileYoureHere/Assets/Scripts/make_a_fire/StoveBehaviour.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using chore;
+using Interactable.Holdable;
+using Mono.Cecil.Cil;
+using ScriptableObjects.Events;
+using UnityEngine;
+using UnityEngine.VFX;
+
+namespace make_a_fire
+{
+    public class StoveBehaviour : MonoBehaviour
+    {
+        [Header("Burning Objects")] 
+        [SerializeField] private HoldableObjectBehaviour newspaper;
+        [SerializeField] private List<HoldableObjectBehaviour> logs;
+        
+        [Header("Fire Behaviour")] 
+        [SerializeField] private VisualEffect fireParticle;
+        [SerializeField] private AudioClip matchStrike;
+        [SerializeField] private AudioClip burningFire;
+        [SerializeField] private AudioClip chargedFire;
```

Comment:
```suggestion
        [SerializeField] private AudioClip[] fireSounds;
```
Also, you should probably store these inside a Scriptable Object :)